packages/dx-react-chart-bootstrap4/src/index.js

```diff
@@ -0,0 +1,13 @@
+import '../../dx-styles/dx-chart-bs4.css';
+
+export { Chart } from './chart';
+export { Grid } from './plugins/grid';
+export { Legend } from './plugins/legend';
+export { BarSeries } from './plugins/bar-series';
+export { PieSeries } from './plugins/pie-series';
+export { ValueAxis } from './plugins/value-axis';
```

Comment:
Can we reorder these `exports`? I mean, export series, then axes etc.

Reply:
Please clarify, what do you mean? Do you want me to group series, axes and other components?

Reply:
```
export { *Series }
... export other series
export { *Axis }
... export other axes
```